test/snapp_base.js

```diff
@@ -787,7 +787,7 @@ contract("SnappBase", async (accounts) => {
       )
     })
 
-    it("Generic claim", async () => {
+    it.only("Generic claim", async () => {
```

Comment:
please take the only out

Reply:
Oh yes of course sorry! I will push this as a "hot fix" since its already been merged!